Userland/Utilities/mv.cpp

```diff
@@ -22,12 +22,14 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     // NOTE: The "force" option is a dummy for now, it's just here to silence scripts that use "mv -f"
     //       In the future, it might be used to cancel out an "-i" interactive option.
     bool force = false;
+    bool no_clobber = false;
     bool verbose = false;
 
     Vector<String> paths;
 
     Core::ArgsParser args_parser;
     args_parser.add_option(force, "Force", "force", 'f');
+    args_parser.add_option(no_clobber, "Do not overwrite existing files", "no-clobber", 'n');
```

Comment:
Since force is a dummy flag does `-n -f` do what you expect?

Reply:
I haven't implemented that behavior. It seems like BSD and GNU both use the rule that `i`, `f`, and `n` are mutually exclusive flags, and only the last one takes effect. Do you need me to add this logic before merging?

Reply:
I think something (abnormal) should happen when you pass `-f` and `-n`.
But just printing a warning or seems good enough, if the order thing is actually something ports or people expect than we can look at it.

Reply:
I've added a warning that -f overrides -n (regardless of flag order), and removed the comment about -f being unused.